### DISCUSSION TITLE
Apt debian backports

### DISCUSSION THREAD

**Discussion by dpavlic** [pyinfra-dev/pyinfra#739] [Q&A] [answered]
Hello,

This is probably already possible in pyinfra but I'm not sure how. How would I got about specifying a backports version of a package in apt? Not a specific version, just prefer the latest backports version for one specific package?

**Comment by uggedal** [comment, marked as answer]
This works:

```py
apt.packages(
    name="Install git from backports",
    packages=["git"],
    extra_install_args="-t bullseye-backports",
)
```
